client/src/components/SingleNews/style.scss

```diff
@@ -0,0 +1,54 @@
+.single-news {
+  position: relative;
+  height: 100px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  border: 1px solid $gallery;
+
+  &:hover {
+    cursor: url('../../assets/img/cursor-hover.svg'), auto;
+    background: $white;
+    box-shadow: 0px 2px 24px $news-hover;
+
+    .hover-arrow {
+      display: block;
+      cursor: url('../../assets/img/cursor-hover.svg'), auto;
+    }
+  }
+
+  .hover-arrow {
+    display: none;
```

Comment:
strange

Reply:
i need to show it only on hover :)

Reply:
could u pls add screen  here how it looks

Reply:
<img width="409" alt="123" src="https://user-images.githubusercontent.com/33002574/64069131-29a87f00-cc4c-11e9-985a-577cf96ee210.png">

Reply:
thx